docs/installation-instructions.rst

```diff
@@ -17,7 +19,7 @@ Paste the following URL into the *Additional boards manager URLs* input field un
 
 ::
 
-    https://github.com/Infineon/arduino-core-psoc/releases/latest/download/package_infineon_index.json
+    https://github.com/Infineon/arduino-core-psoc/releases/latest/download/package_psoc_index.json
```

Comment:
Do we want to rename this package_infineon_index to package_psoc_index?

Reply:
I think we cannot have the same name for both xmc-for-arduino and arduino-core-psoc... But I am not sure. 
So i would say we name it only as psoc, and maybe at one point the other is named after xmc or?

Reply:
Yes we can try that